packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -492,24 +540,34 @@ class SingleActivator with Diagnosticable, MenuSerializableShortcut implements S
   ///
   /// If [includeRepeats] is true, the activator is checked on all
   /// [RawKeyDownEvent] events for the [trigger] key. If [includeRepeats] is
-  /// false, only [trigger] key events with a false [RawKeyDownEvent.repeat]
-  /// attribute will be considered.
+  /// false, only [trigger] key events which are [KeyRepeatEvent]s will be
+  /// considered.
```

Comment:
I'm wondering if we can rework repeated events as a kind of stateful activators, which also eliminates the use of `triggers`.

Reply:
I think changing that is out of scope here. We can do that in another PR if we decide to.